widget/embedded/src/components/Layout/Layout.tsx

```diff
@@ -73,12 +91,23 @@ function LayoutComponent(props: PropsWithChildren<PropTypes>, ref: Ref) {
         }
       />
       <Content noPadding={noPadding}>{children}</Content>
+      <ActivateTabModal
+        open={showActivateTabModal}
+        onClose={() => setShowActivateTabModal(false)}
+        onConfirm={onActivateTab}
+      />
       {(hasLogo || footer) && (
         <Footer>
           <div className="footer__content">{footer}</div>
           {hasLogo && (
             <div className="footer__logo">
               <Divider size={12} />
+              {isActiveTab === false && (
```

Comment:
If a page has set `hasLogo: false` the alert shouldn't be shown. I don't think this is desired behavior.